nifi-nar-bundles/nifi-cdc/nifi-cdc-mysql-bundle/nifi-cdc-mysql-processors/src/main/java/org/apache/nifi/cdc/mysql/processors/CaptureChangeMySQL.java

```diff
@@ -681,6 +731,9 @@ protected void connect(List<InetSocketAddress> hosts, String username, String pa
                 binlogClient.setBinlogPosition(currentBinlogPosition);
             }
 
+            binlogClient.setGtidSet(currentGtidSet);
+            binlogClient.setGtidSetFallbackToPurged(true);
```

Comment:
Is it what we want here? https://github.com/shyiko/mysql-binlog-connector-java/blob/master/src/main/java/com/github/shyiko/mysql/binlog/BinaryLogClient.java#L331

Reply:
@pvillard31 We need it to realize the following:

- The processor can get the binlog even if the binlog is purged on the connected host.
- When reconnecting to another host, the processor can get the binlog even if the binlog is already purged on that host.

Reply:
Understood